package.json

```diff
@@ -12,6 +12,10 @@
     "type": "git",
     "url": "https://github.com/t3-oss/create-t3-app.git"
   },
+  "workspaces": [
```

Comment:
This is not needed since we are using pnpm workspaces: https://pnpm.io/pnpm-workspace_yaml